### ISSUE TITLE
Xcode breaks on performSegueWithIdentifier:sender:

### ISSUE THREAD

**Issue by jpcguy89** [John-Lluch/SWRevealViewController#726]
SWRevealViewController works amazingly. I just have one minor irritation I would like to see fixed. Every time I run my app, if I have breakpoints enabled it always breaks twice on line 1713 in SWRevealViewController.m 

`[self performSegueWithIdentifier:SWSegueRightIdentifier sender:nil];`

It's not a fatal error, and I'm not sure why Xcode is throwing an exception, but it is.

Thanks!

**Comment by iDevelopper** [comment]
I think you have to clear the project (Menu: Product->Clean and Alt+Product->Clean Build Folder...)

**Comment by jpcguy89** [comment]
Nope. Followed the exact instructions. Still breaks there.

**Comment by jpcguy89** [comment]
<img width="1180" alt="breakpointswrevealviewcontroller" src="https://user-images.githubusercontent.com/1883268/29742815-8d2d61ac-8a4b-11e7-9d2e-c07cbf35a686.png">

**Comment by iDevelopper** [comment]
Verify the segue identifier in the storyboard. It should be sw_front or sw_rear or sw_right

**Comment by jpcguy89** [comment]
Clarification: You're saying all segues that are going from the "side menu" should have "sw_right" as the segue identifier?

I have "sw_rear" set for the side menu, and "sw_right" set for the initial view controller. The rest have their own unique identifiers.

**Comment by iDevelopper** [comment]
You must have a sw_front segue for the front view controller. sw_rear is for the left side controller. sw_right for the right side controller.

**Comment by jpcguy89** [comment]
I apologize, I misspoke. The segue to the initial controller is, in fact, sw_front, not sw_right.

**Comment by iDevelopper** [comment]
And then? Always the issue? If so, could you upload a project?

**Comment by jpcguy89** [comment]
Every time. I wouldn't feel comfortable posting the project publicly. I know that's not helpful, but I've been working on this for years and can't even fathom if someone were to publish an app using the code I've written.

**Comment by iDevelopper** [comment]
Ok, but perhaps can you reproduce the same issue in a sample project?

**Comment by jpcguy89** [comment]
Yep, was able to reproduce the issue with a new test project. Added a breakpoint to break on all exceptions, ran, and sure enough, it hit that same line of code.
[RevealViewControllerTest.zip](https://github.com/John-Lluch/SWRevealViewController/files/1264301/RevealViewControllerTest.zip)

**Comment by iDevelopper** [comment]
Added a sw_front set segue to the blue view controller:

[RevealViewControllerTest.zip](https://github.com/John-Lluch/SWRevealViewController/files/1265962/RevealViewControllerTest.zip)

**Comment by jpcguy89** [comment]
Any news on this? I believe the issue was still there after you added the segue.

**Comment by iDevelopper** [comment]
No, I do not have this issue!

**Comment by jpcguy89** [comment]
And you added a breakpoint for all obj-c and C++ exceptions?

**Comment by iDevelopper** [comment]
No, why do you want to add a breakpoint?

**Comment by iDevelopper** [comment]
Ok I see, I added a breakpoint. It is normal because there is no segue with identifier sw_right in the application. But the exception is not thrown. So, no crash...

There is no documented method in Apple Documentation to check if a segue exist, so the author used a workaround!
